inc/Lane.hpp

```diff
@@ -9,4 +10,5 @@ class Lane
 public:
     Lane();
     int getNumberOfGames();
+    std::ifstream open(std::string pathToFile);
```

Comment:
To są zmiany Kasi, i one najpierw powinny zostać zmergowane i dopiero wtedy powinnaś swoje zadanie dostarczać.